src/components/modals/modalWrapper.tsx

```diff
@@ -1,12 +1,20 @@
-import React from "react";
 import * as Dialog from "@radix-ui/react-dialog";
-import { motion, AnimatePresence } from "framer-motion";
+
+import React from "react";
 import Image from "next/image";
+
 import { useModal } from "@/contexts/ModalContext";
+import { motion, AnimatePresence, color } from "framer-motion";
+import {
+  largeShadowColors,
+  LargeShadowColorOption,
+} from "@/lib/stylingData/largeShadowColors";
 
 interface ModalWrapperProps {
   children: React.ReactNode;
   modalKey: number;
+  variant?: "filled" | "hollow";
```

Comment:
I would delete this since we're not using it. (and colorScheme below). The functionality is set up for it to actually work